tests/test_cookies.py

```diff
@@ -50,6 +50,13 @@ def on_put(self, req, resp):
         resp.unset_cookie("bad")
 
 
+class CookieResourceMaxAgeFloat:
+
+    def on_get(self, req, resp):
```

Comment:
Could you also add a test for setting max_age to a string value? Since that now works, apps may come to rely on it and it would be helpful to know if the implementation ever breaks that contract (implicit though it is).